ui/src/frontend/panel_container.ts

```diff
@@ -237,32 +239,60 @@ export class PanelContainer implements m.ClassComponent<Attrs> {
     return (attrs as {collapsed?: boolean}).collapsed !== undefined;
   }
 
-  renderPanel(node: AnyAttrsVnode, key: string, extraClass = ''): m.Vnode {
+  renderPanel(node: AnyAttrsVnode, key: string, style?: object): m.Vnode {
     assertFalse(this.panelByKey.has(key));
     this.panelByKey.set(key, node);
 
     return m(
-        `.panel${extraClass}`,
-        {key, 'data-key': key},
+        `.panel`,
+        {key, 'data-key': key, style},
         perfDebug() ?
             [node, m('.debug-panel-border', {key: 'debug-panel-border'})] :
             node);
   }
+  getHeightOfParent(id?: string):number {
```

Comment:
nit: Please put a blank line between functions/methods.
```suggestion

  getHeightOfParent(id?: string):number {
```